src/examples/operational-apis-demo.ts

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash'
```

Comment:
the only thing you use is the `isEmpty`, should we just import that one specifically?